ArduPlane/system.cpp

```diff
@@ -39,7 +39,7 @@ void Plane::init_ardupilot()
 #if AP_RANGEFINDER_ENABLED
     // initialise rangefinder
     rangefinder.set_log_rfnd_bit(MASK_LOG_SONAR);
-    rangefinder.init(ROTATION_PITCH_270);
+    rangefinder.init(rangefinder_orientation());
```

Comment:
I don't think this is a good idea. This value is used to set the param defaults. For example if I had a existing rangefinder and then added a second one. I would set that second one to a custom orientation and then set the new `RNGFND_LND_ORNT` to that custom rotation. However on reboot I would find my original rangefinder has now also changed to that orientation and I would have to go and manually set it back to down.

Reply:
yep, good catch! fixed